src/reducers/versions.tsx

```diff
@@ -195,7 +195,7 @@ type VersionAddon = {
   iconUrl: string;
   id: number;
   name: LocalizedStringMap;
-  slug: string;
+  slug: string | null;
```

Comment:
It would be nice to update the addons-server docs too: https://addons-server.readthedocs.io/en/latest/topics/api/reviewers.html#get--api-v4-reviewers-addon-(int-addon_id)-versions-(int-version_id)-

Reply:
https://github.com/mozilla/addons-server/pull/14963

Reply:
I had a further thought about this. If we didn't want to allow `null` in `VersionAddon`, we could change `createInternalVersion` to that it stores an empty string in `slug` if it finds a `null` in the response. Doing that would eliminate the need for the ternary added to `src/components/Navbar/index.tsx` above. The downside is that we would no longer be able to tell the difference between a slug with a value of `''` and a slug with a value of `null`, so it might be better to just leave it like this.

@willdurand WDYT?

Reply:
I'd prefer to have the `slug` declared as nullable like it's done here. @eviljeff confirmed that we have `null` slugs for deleted add-ons, which is something we definitely need to support in a tool designed for reviewers.

As you said, using an empty string would be confusing because this isn't allowed.

Reply:
Thanks @willdurand. In case it isn't entirely clear, @wagnerand, this means just leave it as it is now.